### PR TITLE
Feat destroy wrapper method

### PR DESCRIPTION
**Pull request by vouill**
This PR adds

- `.destroy()` to `Wrapper`.

Heavily inspired from avoriaz :)

### CODE REVIEW

File: src/wrappers/error-wrapper.js [hunk 1]
```diff
@@ -99,4 +99,8 @@ export default class ErrorWrapper implements BaseWrapper {
   update (): void {
     throwError(`find did not return ${this.selector}, cannot call update() on empty Wrapper`)
   }
+
+  destroy (): void {
+    throwError(`find did not return ${this.selector}, cannot call update() on empty Wrapper`)
```
Comment (eddyerburgh) [inline src/wrappers/error-wrapper.js]: should be `... cannot call destroy()`

File: test/unit/specs/wrappers/error-wrapper.spec.js [hunk 2]
```diff
@@ -154,4 +154,11 @@ describe('ErrorWrapper', () => {
     const error = new ErrorWrapper(selector)
     expect(() => error.update()).to.throw().with.property('message', message)
   })
+
+  it('destroy throws error when called', () => {
+    const selector = 'div'
+    const message = `[vue-test-utils]: find did not return ${selector}, cannot call update() on empty Wrapper`
```
Comment (eddyerburgh) [inline test/unit/specs/wrappers/error-wrapper.spec.js]: should be `cannot call destroy()`

**Comment by vouill** [comment]
@eddyerburgh PR updated !

**Comment by vouill** [comment]
Alright, thanks for walking me through this ! PR updated !

**Pull request merged**